boards/arm64/imx8mm_evk/imx8mm_evk_defconfig

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_IMX8M_A53=y
+CONFIG_SOC_IMX8MM_A53=y
+CONFIG_BOARD_IMX8MM_EVK_A53=y
+CONFIG_ARM_ARCH_TIMER=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
```

Comment:
A general comment about this file: several options here, such as `CONFIG_XIP=n`, `CONFIG_SMP=y` seem like they should be defined at the SOC level, not the board level

Reply:
This is an initial support for imx8mm arm64 and follow qemu arm64. I would leave this as improvment in future patches.